tcms/bugs/tests/test_admin.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+
+'''
+This module contains tests for the Admin view ('Everything' else page)
+'''
```

Comment:
This isn't necessary but if you insist on having it then make sure pylint is happy.